packages/dds/shared-object-base/src/serializer.ts

```diff
@@ -65,14 +63,8 @@ export interface IFluidSerializer {
 export class FluidSerializer implements IFluidSerializer {
 	private readonly root: IFluidHandleContext;
 
-	public constructor(
-		private readonly runtime: Pick<
-			// eslint-disable-next-line import/no-deprecated
-			IFluidDataStoreRuntimeExperimental,
-			"inStagingMode" | "channelsRoutingContext"
-		>,
-	) {
-		this.root = this.runtime.channelsRoutingContext;
+	public constructor(private readonly context: IFluidHandleContext) {
```

Comment:
nit: Just to not confuse this with other contexts that we have.
```suggestion
	public constructor(private readonly handleContext: IFluidHandleContext) {
```

Reply:
This is reverting back to how it is in main so not going to change it.  Going to merge this whole test branch into main next.